mentat/command/commands/clear.py

```diff
@@ -32,4 +34,6 @@ def argument_autocompletions(
     @override
     @classmethod
     def help_message(cls) -> str:
-        return "Clear the current message history."
+        return (
+            "Clear the current message history and auto included features from context."
```

Comment:
"code features" instead of "features" as users won't know what a feature is

```suggestion
            "Clear the current message history and auto included code features from context."
```